cpp/src/io/utilities/column_stats.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
"stats" means something very different for ORC and Parquet. I'd change the name of this file to reflect that it's about data type inference.